examples/kcl-apply.yaml

```diff
@@ -0,0 +1,10 @@
+functionConfig:
```

Comment:
The field `functionConfig:` is required? May I omit it?

Reply:
Must need the field `functionConfig` from my test.